README.md

```diff
@@ -21,3 +21,42 @@ $ make
 
 [nasm]: https://www.nasm.us/
 [qemu]: https://www.qemu.org/
+
+## Building on MacOS
+
+The following instructions have only been tested on Mojave 10.14.5
+using Xcode 10.2.
+
+### Setting up your Mac as a dev machine
+
+Your life will be overall easier if you do the following. You only
+have to do these steps once on your Mac to make it a "developer
+machine".
+
+1. Install Xcode 10.2 from the AppStore. If you don't want to install
```

Comment:
Why do we need Xcode?

Reply:
Will update PR to remove the "novice" help guide.